crates/datafusion_ext/src/planner/expr/mod.rs

```diff
@@ -26,23 +26,41 @@ mod substring;
 mod unary_op;
 mod value;
 
-use crate::planner::{AsyncContextProvider, SqlQueryPlanner};
 use async_recursion::async_recursion;
 use datafusion::arrow::datatypes::DataType;
 use datafusion::common::tree_node::{Transformed, TreeNode};
 use datafusion::common::{Column, DFSchema, DataFusionError, Result, ScalarValue};
-use datafusion::logical_expr::expr::ScalarFunction;
-use datafusion::logical_expr::expr::{InList, Placeholder};
+use datafusion::logical_expr::expr::{InList, Placeholder, ScalarFunction};
 use datafusion::logical_expr::{
-    col, expr, lit, AggregateFunction, Between, BinaryExpr, BuiltinScalarFunction, Cast, Expr,
-    ExprSchemable, GetFieldAccess, GetIndexedField, Like, Operator, TryCast,
+    col,
+    expr,
+    lit,
+    AggregateFunction,
+    Between,
+    BinaryExpr,
+    BuiltinScalarFunction,
+    Cast,
+    Expr,
+    ExprSchemable,
+    GetFieldAccess,
+    GetIndexedField,
+    Like,
+    Operator,
+    TryCast,
```

Comment:
Not a big deal as the imports are easily folded, but I prefer having blocks instead of splitting every item into a newline

Reply:
yeah, I agree that there are cases where it's a bit extreme, but in most cases I think the expanded form ends up being more readable, and easier to grep.